SECURITY.MD

```diff
@@ -0,0 +1,35 @@
+<!-- BEGIN FLEDGE SECURITY.MD V0.0.1 BLOCK -->
+
+## Security
+
+FLEDGE takes the security of our software products and services seriously, which includes all source code repositories managed through our GitHub organizations, which include [FLEDGE](https://github.com/fledge-iot)
+
+
+If you believe you have found a security vulnerability in any Fledge-owned repository please report it to us as described below.
+
+## Reporting Security Issues
+
+**Please do not report security vulnerabilities through public GitHub issues.**
```

Comment:
Instead, send an email at the following given address.

Reply:
Moved the email address into this message instead of having it on the next line.